sentry-apollo/src/test/java/io/sentry/apollo/SentryApolloInterceptorTest.kt

```diff
@@ -30,8 +33,9 @@ class SentryApolloInterceptorTest {
 
     class Fixture {
         val server = MockWebServer()
+        val hub = spy(HubAdapter.getInstance())
```

Comment:
out of curiosity, why `spy` instead of `mock`?

Reply:
Initially it was meant to be full integration test, using coroutines Sentry context. But, since on Android Sentry context does not need to be used, I've changed this test to use mocked hub instead of verifying transport.